src/elements/common/messages.ts

```diff
@@ -1,9 +1,3 @@
-/**
- * @flow
- * @file i18n messages
- * @author Box
- */
-
```

Comment:
what was this @flow tag used for?

Reply:
for flowtypes, when you convert a js to a js.flow this will support the flowtype imports allowing the tsx files to support typescript imports